spec/controllers/api/api_stages_controller_spec.rb

```diff
@@ -140,12 +140,11 @@
     data = { attributes: { instructions: "#{Faker::Lorem.paragraph}",
                            name: 'hello',
                            order: 0 },
-             actions: [{ name: "#{Faker::Lorem.word}",
-                         description: "#{Faker::Lorem.paragraph}" }],
+             actions: [{ name: "" }],
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.